apps/src/metrics/MetricsReporter.ts

```diff
@@ -79,6 +79,9 @@ class MetricsReporter {
 
   /**
    * Publish a metric.
+   *
+   * Note that this will send two metrics, with and without the browser version dimension.
```

Comment:
Nit: maybe could include that browser name is included in both metrics.

Reply:
Good point, will merge to get started on alarms but can revisit this documentation later.